### PR TITLE
feat: excel like paste when paste cell range is larger than input range

### PR DESCRIPTION
**Pull request by yhay81**
Hello,

I've opened a PR same as https://github.com/jspreadsheet/ce/pull/594, targeting a feature I believe is crucial. Eager to contribute more, I welcome any guidance on how I can assist further with this feature or elsewhere in the project.

Your feedback would be greatly appreciated. Thank you for your hard work on this project!

### CODE REVIEW

**Comment by yhay81** [comment]
Related https://github.com/jspreadsheet/ce/issues/591

File: src/index.js [hunk 1]
```diff
@@ -6451,8 +6451,24 @@ if (! formula && typeof(require) === 'function') {
          * @param integer row number
          * @return string value
          */
-        obj.paste = function(x, y, data) {
+        obj.paste = function(selectedCell, data) {
             // Paste filter
+            selectedCell = selectedCell.map((cell) => parseInt(cell));
```
Comment (yhay81) [inline src/index.js]: This line is the only difference from https://github.com/jspreadsheet/ce/pull/594.
Upon debugging, I found a singular bug: the expression `selectedCell[2] < selectedCell[0]` evaluates to true when `selectedCell[2]` and `selectedCell[0]` are string values like `"10"` and `"2"`, respectively, due to string comparison.

**Comment by yhay81** [comment]
Hello @hodeware and @giko45,

I've initiated this PR echoing the valuable effort behind @giko45's #594, focusing on a vital feature for me. I'm fully prepared to make any required updates to align with the project's current needs.

@Giko45, your foundational work is deeply appreciated. If you're still passionate about this feature and willing, your support in engaging with @hodeware would mean a lot.

**Comment by hodeware** [comment]
Hi @yhay81, thanks a lot. I would be happy to review your PR. There is one change required.

The method signature should stay the same. The compatibility with all versions and distributions are very important.
obj.paste = function(x, y, data);

Solution: it seems you need the selection range inside the paste, you can use the following property which is already available on the scope of the function.
obj.selectedCell

It would be great if you can commit those changes, so we can proceed with the merge.

**Comment by giko45** [comment]
Hi @yhay81,

Happy it comes into use :).

I don't have the code readly available but try as @hodeware suggest: just change your extra line to:
`selectedCell = obj.selectedCell.map((cell) => parseInt(cell));`
and rollback all jexcel.current.paste calls.

**Comment by hodeware** [comment]
Thanks, we are reviewing the PR soon.

**Comment by yhay81** [comment]
@hodeware, @giko45
Thank you for the feedback. I've made the changes using obj.selectedCell and committed them!

**Review by hodeware — APPROVED** [review]

**Comment by hodeware** [comment]
Thanks

**Pull request merged**